scripts/lint.sh

```
@@ -0,0 +1,14 @@
+#!/bin/sh
+for dir in $(ls -d charts/*/); do
```

Shellcheck (0.10.0)

[warning] Iterating over ls output is fragile. Use globs.

(SC2045)